src/main/java/ch/njol/skript/lang/globals/GlobalOptions.java

```diff
@@ -0,0 +1,68 @@
+package ch.njol.skript.lang.globals;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.Config;
+import ch.njol.skript.structures.StructOptions;
+import ch.njol.util.StringUtils;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+
+/**
+ * Represents the 'globals/options.sk' file
+ */
+public class GlobalOptions extends GlobalFile {
+
+	private static final Map<String, String> options = new HashMap<>();
```

Comment:
What do you think about instead having an OptionsRegistry on the Skript instance (the one from org.skriptlang.skript)? This organization could remain mostly the same, but the `load` method could instead take in an `OptionsRegistry` instance and load it into there. This would also enable addons to define their own global options.

This would also make the test script easier.